x-pack/test/search_sessions_integration/tests/apps/discover/sessions_in_space.ts

```diff
@@ -130,11 +131,12 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
 
       after(async () => {
+        await PageObjects.security.forceLogout();
```

Comment:
I only have one piece of feedback:
Since this has been a common oversight, and it could easily happen again in the future, how about we change all of these to add a comment?

```suggestion
        // NOTE: Logout needs to happen before anything else to avoid flaky behavior
        await PageObjects.security.forceLogout();
```

Reply:
Yeah, that's a good idea, will add comments!

Reply:
Added where it makes sense (in cleanup), it will cost us 9+ more reviews from the affected teams :slightly_smiling_face: